src/Term/TermFallback.php

```diff
@@ -46,6 +46,18 @@ public function __construct( $requestedLanguageCode, $text, $actualLanguageCode,
 			throw new InvalidArgumentException( '$sourceLanguageCode must be a non-empty string or null' );
 		}
 
+		if ( $actualLanguageCode === $sourceLanguageCode ) {
```

Comment:
Why should the user of this class be deprived of the notification that they did something wrong in favor of silently doing something that maybe was not intended?